packages/wonder-blocks-button/components/button.md

```diff
@@ -363,13 +363,14 @@ const styles = StyleSheet.create({
 </View>
 ```
 
-Layouts often specify a specific width of button.  When implementing such
-designs use `minWidth` instead of `width`.  `minWidth` allows the button
-to resize to fit the content whereas `width` does not.  This is important
+Layouts often specify a specific width of button. When implementing such
+designs use `minWidth` instead of `width`. `minWidth` allows the button
+to resize to fit the content whereas `width` does not. This is important
 for international sites since sometimes strings for UI elements can be much
-longer in other languages.  Both of the buttons below have a "natural" width
-of 144px.  The one on the right is wider but it accommodates the full string
-instead of wrapping or truncating it.
+longer in other languages. Both of the buttons below have a "natural" width
+of 144px. The one on the right is wider but it accommodates the full string
+instead of wrapping it. Try to avoid setting a `maxWidth`, but if one is set,
+the text truncates.
```

Comment:
It's probably worth calling out that the text will truncate as well if the container the buttons are in doesn't have enough room to accommodate the natural width.  If you view this example on mobile, the text in the right button will be truncated even without `maxWidth`.  We should probably drop the part about `maxWidth`.